<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QMainWindow" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main Window Background */
QWidget {
    background-color: #121212;
    color: #e0e0e0;
    font-family: 'Poppins', 'Segoe UI', sans-serif;
    font-size: 15px;
    letter-spacing: 0.4px;
    padding: 10px;
}

/* Labels */
QLabel {
    font-size: 17px;
    color: #ffffff;
    font-weight: 500;
}

/* Buttons */
QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ff5c93, stop:1 #ffc371);
    border: none;
    border-radius: 14px;
    color: white;
    font-size: 15px;
    font-weight: bold;
    padding: 10px 22px;
    min-width: 120px;
    box-shadow: 0px 3px 8px rgba(0, 0, 0, 0.2);
    transition: all 0.25s;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ffc371, stop:1 #ff5c93);
    box-shadow: 0px 5px 12px rgba(255, 92, 147, 0.5);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #e34e82, stop:1 #ff9f57);
    transform: translateY(2px);
}

/* ComboBox */
QComboBox {
    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    padding: 6px 12px;
    font-size: 14px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 18px;
    background-color: #ff5c93;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

QComboBox QAbstractItemView {
    background-color: #1f1f1f;
    color: #ffffff;
    selection-background-color: #ff5c93;
    border-radius: 8px;
}

/* LineEdit */
QLineEdit {
    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    border-radius: 10px;
    color: white;
    font-size: 14px;
    padding: 8px 12px;
}

/* Checkboxes */
QCheckBox {
    color: #ffffff;
    font-size: 14px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border: 2px solid #ff5c93;
    border-radius: 4px;
    background-color: transparent;
    transition: background-color 0.25s;
}

QCheckBox::indicator:checked {
    background-color: #ff5c93;
    border: 2px solid #ff9f57;
}

/* ScrollBar */
QScrollBar:vertical {
    background: #1f1f1f;
    width: 12px;
    margin: 4px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    background: #ff5c93;
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::add-line, QScrollBar::sub-line {
    background: none;
}

/* Table Widget */
QTableWidget {
    background-color: #1f1f1f;
    border: 2px solid #ff5c93;
    color: #ffffff;
    font-size: 14px;
    gridline-color: #ff5c93;
}

QHeaderView::section {
    background-color: #ff5c93;
    color: white;
    padding: 6px;
    border: none;
}

/* Tooltips */
QToolTip {
    background-color: #ff5c93;
    color: white;
    padding: 6px;
    border-radius: 8px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_company">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPANY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
